modules/apps/transfer/keeper/mbt_relay_test.go

```diff
@@ -322,7 +322,7 @@ func (suite *KeeperTestSuite) TestModelBasedRelay() {
 			description := fileInfo.Name() + " # " + strconv.Itoa(i+1)
 			suite.Run(fmt.Sprintf("Case %s", description), func() {
 				seq := uint64(1)
-				packet := channeltypes.NewPacket(tc.packet.Data.GetBytes(), seq, tc.packet.SourcePort, tc.packet.SourceChannel, tc.packet.DestPort, tc.packet.DestChannel, clienttypes.NewHeight(1, 100), 0)
+				packet := channeltypes.NewPacket([]byte("mockdata"), seq, tc.packet.SourcePort, tc.packet.SourceChannel, tc.packet.DestPort, tc.packet.DestChannel, clienttypes.NewHeight(1, 100), 0)
```

Comment:
why did this change?

Reply:
Because the type here is no longer a proto type, it doesn't have that method. And the packet is not actually using the data here, so it doesn't matter what the data is.